types/make-fetch-happen/make-fetch-happen-tests.ts

```diff
@@ -72,10 +73,16 @@ fetcher('http://url', { proxy: new URL('http://secure-proxy') });
 // $ExpectType Promise<Response>
 fetcher('http://url', { proxy: new NodeURL('http://secure-proxy') });
 
-// Test the imported `tls` type `rejectUnauthorized` remapped to `strictSSL`.
+// Test the imported `tls` type `CommonConnectionOptions.rejectUnauthorized` remapped to `strictSSL`.
 // $ExpectType Promise<Response>
 fetcher('https://url', { strictSSL: true });
 
+const options: CommonConnectionOptions = {
+    rejectUnauthorized: true
```

Comment:
Adding this to tests so that we could catch and potential changes to the type of `CommonConnectionOptions. rejectUnauthorized` as that is ideally where the type comes from.